content/educators/yung-qi.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Yung-Qi
+pronouns_en: he/him
+pronouns_nl: hij/hem
+quote:
+  Ik probeer niemand te overtuigen om anderen te accepteren die zich zelf zijn, maar ik probeer wel je horizon te verbreden om anderen lief te hebben zoals die zijn.
```

Comment:
```suggestion
  Ik wil niemand overtuigen, maar ik probeer wel je horizon te verbreden om anderen lief te hebben zoals die zijn.
```
(oid)

Anders wordt ie wel erg lang
![CleanShot 2024-04-11 at 19 34 57@2x](https://github.com/dwh-outsite/dwhdelft.nl/assets/15815208/bf8be083-3836-4748-83be-4a4309dd98e0)

Reply:
Ik heb er nu 'Ik wil niemand te overtuigen om anderen te accepteren, maar ik probeer wel je horizon te verbreden om anderen lief te hebben zoals ze zijn.' van gemaakt, iets meer true to the quote, en even lang als Dante's quote